src/test/resources/metadata-manager/inventory_instance_with_2_items.json

```diff
@@ -1,6 +1,6 @@
 {
   "instanceid": "2f7f23be-81a6-4227-85ba-7c1c81272ee1",
-  "instanceupdateddate": "2020-06-01T10:56:10.118Z",
+  "updateddate": "2020-06-01T10:56:10.118Z",
```

Comment:
I must be missing some detail here... why aren't these camelCase?

Reply:
We have a database view which returns data in such manner, that's why these fields are named in such way.

Reply:
> We have a database view which returns data in such manner
why?

Reply:
Well, it's in another module(which is already released), we have to integrate with it. If you think this should be changed, I suggest we do it separately.

Reply:
What I'm seeing in the schemas is that these fields are camelCase... Am I looking at the wrong APIs?

https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/oai-pmh-view.html#

Reply:
correction - I see that there are a few fields which aren't camelCase... I'm really curious why this was done as it's inconsistent, not only within the schema but across all of FOLIO

Reply:
I'm looking at this PR:  https://github.com/folio-org/mod-inventory-storage/pull/473 and the schemas use camelCase, but when I actually make calls to the API (folio-testing) they're all lowercase.

Reply:
We're using
https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/oai-pmh-view.html#oai_pmh_view_updatedinstanceids_get
and
https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/oai-pmh-view.html#oai_pmh_view_enrichedinstances_post

Reply:
Yes, when a call is made to API, the values are in lower case

Reply:
right... and look at the response schema for those APIs... they have "instanceId" (camelCase).  but when you call the actual API you mysteriously get "instanceid" (all lowercase).  I'm confused what's going on here